configs/katello/4.9.yaml

```diff
@@ -105,6 +105,12 @@
   :4.9.0:
     :redmine_version_id: 1645
 :prior_releases:
+  :4.8.4:
```

Comment:
For my understanding, is 4.8.4 a prior release? 4.9.0 was released prior to 4.8.4 (which isn't released yet) so does it matter? Or is it defined as all 4.8 versions?

If so, can't you just query the Redmine API at the time of calling? Just open https://projects.theforeman.org/projects/katello/versions.json and filter the name on `Katello 4.8.*`.